public/Lab-Files/Lesson02/Challenges/text-level-element.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Text Level Elements</title>
+</head>
+
+<body>
+    <a href="/">Go Back</a>
+    <p>Web development is a broad term for the work involved in developing a web site for the Internet <strong>(World Wide Web) or an intranet (a private network)</strong>. Web development can range from developing the simplest static single page of plain
+        text to the most complex web-based internet applications, electronic businesses, and social network services. A more comprehensive list of tasks to which <em>web development commonly refers, may include web engineering, web design, web content development, client
+        liaison, client-side/server-side scripting, web server and network security configuration, and e-commerce development</em>. Among web professionals, "web development" usually refers to the main non-design aspects of building web sites: writing
+        markup and coding. Most recently Web development has come to mean the creation of <strong>content management systems or CMS</strong>. These CMS can be made from scratch, proprietary or open source. In broad terms the CMS acts as middleware between
+        the database and the user through the browser. A principle benefit of a CMS is that it allows non-technical people to make changes to their web site without having technical knowledge.</p>
+</body>
+
```

Comment:
Awesome job with these first challenges, Johnny!

Your code is looking good, and I can tell that you have been working hard, and your code reflects that.
This is just nitpicking, but the only thing I'd suggest is that you be more consistent with how you space each element.

For example, if you normally don't have a blank line between the ``` <head> ``` and the ```<body>```, then don't add it. Try to keep your code consistent. other than that, you are doing great. Keep it up!